cupy/cuda/compiler.py

```diff
@@ -89,21 +89,23 @@ def _run_cc(cmd, cwd, backend, log_stream=None):
 
 @_util.memoize()
 def _get_extra_path_for_msvc():
-    import distutils.spawn
-    cl_exe = distutils.spawn.find_executable('cl.exe')
+    import shutil
```

Comment:
Could you move this import (and `import platform` too --- as they're standard modules) to the top of the file?